### ISSUE TITLE
Ctrl+C in the "payment sent" window

### ISSUE THREAD

**Issue by rmrm-rami** [Feathercoin-Foundation/electrum-ftc#17]
It's not possible to select the tx ID from the confirmation window:

![image](https://user-images.githubusercontent.com/37974898/38490536-a5ee316c-3be9-11e8-98bc-b3e14d4baafc.png)

It would be nice if the tx ID had an underlying hyperlink directing to the blockchain explorer directly so one can check it there as well, if wanted. Maybe even to the new one: https://chainz.cryptoid.info/ftc/

**Comment by lclc** [comment]
Can't reproduce on OSX:
![image](https://user-images.githubusercontent.com/1846033/38614853-ec81f108-3d8e-11e8-8573-8eabb8e95bde.png)

**Comment by toxeus** [comment]
I cannot reproduce this on Linux too:
![lala](https://user-images.githubusercontent.com/7702854/38657640-ab4aeeea-3e21-11e8-934e-f82ccf421b57.png)

**Comment by toxeus** [comment]
I was able to reproduce this on Windows. It can be fixed by setting the text interaction explicitly (not tested):
```python
msgbox.setTextInteractionFlags(QtCore.Qt.TextSelectableByMouse)
```

**Comment by toxeus** [comment]
@rmrm-rami wrote:
> It would be nice if the tx ID had an underlying hyperlink directing to the blockchain explorer directly so one can check it there as well, if wanted.

1. click on the `History` tab
1. right-click on desired tx
1. click on `View in block explorer`

@rmrm-rami if you double-click a tx then a window opens where the tx id can be copy&pasted. What is your use case for copy&pasting it from the confirmation message box?

**Comment by rmrm-rami** [comment]
@toxeus Usecase would be if I already have a block explorer open where I could copy&paste it straight from the confirmation message box, without going through the History-tab.

It's just a "nice-to-have" though, not critical.

**Comment by lclc** [comment]
@rmrm-rami did you check if this is also the case with upstream (Bitcoin) Electrum ?

If so, a issue there would make sense as it's not a Feathercoin specific thing.

**Comment by rmrm-rami** [comment]
No, didn't check there. Will try to check when time allows... :-)
